docs/guides/walkthroughs/walkthrough-guides.md

```diff
@@ -0,0 +1,10 @@
+---
+sidebar_position: 1
+description: Walkthough guides.
+---
+
+# Walkthough guides
```

Comment:
I'm in favour having such kind of detailed step-by-step walkthroughs in blogposts, rather than in the official docs. They are highly specific and often quickly changing.

Reply:
We've discussed this with Oisin, Brett, and Kody and agreed to create this new section in the docs for guides.